src/main/java/com/spotinst/sdkjava/SpotinstClient.java

```diff
@@ -117,6 +117,11 @@ public static SpotinstAccountAdminClient getAdminAccountClient(String authToken)
         return spotAdminAccountClient;
     }
 
+    public static SpotinstAccountAdminClient getAdminAccountClient(String authToken, String account) {
```

Comment:
Will this be another client, together with existing client?

Reply:
Yes. It is another client which can be used for account level APIs